internal/controller/server/repository/job/job_test.go

```diff
@@ -88,7 +90,7 @@ func TestCheckJobIsAvailableForEtcdClientFailure(t *testing.T) {
 	mockClient.On("GetValue", "metadata/testJobName").Return("", errors.New("failed to get value from database"))
 
 	jobAvailabilityStatus, err := jobRepository.CheckJobIsAvailable(context.Background(), "testJobName")
-	assert.Equal(t, "failed to get value from database", err.Error())
+	assert.Equal(t, status.Error(codes.Internal, errors.New("failed to get value from database").Error()).Error(), err.Error())
```

Comment:
change errors.New("failed to get value from database").Error() to "failed to get value from database"